collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankFormTest.java

```diff
@@ -611,4 +620,160 @@ public void questions_shouldHavePrefilledValue() {
                 .clickSaveAndExit();
     }
 
+    @Test
+    public void selectedAnswer_shouldBeDisplayed() {
```

Comment:
Please add more context here as well. In this case the value is displayed in another question so it could be described.